### PR TITLE
Update readme to include table changes

### LINKED ISSUE #511: [Table] Phase 1 Feature - Table commandline parameters
- [ ] Readme updates about table commandline parameters descriptions
- [ ] Sanity check every commandline parameter works in
  - [ ] azurite command
  - [ ] azurite-table command

Tips:
Build project then `npm install -g` to install `azurite` and `azurite-table` globally, then do the testing.

### PR DESCRIPTION
**Pull request by ocdi**
I installed my locally build package globally and tested the command line parameters. I updated the README to note Table specific items. Seems to work for me.

Fixes #511

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,12 @@ Start queue service only:
 $ azurite-queue -l path/to/azurite/workspace
 ```
 
+Start table service only:
+
+```bash
+$ azurite-table -l path/to/azurite/workspace
+```
+
```
Comment (blueww) [inline README.md:105]: should also update following section "Following extension configurations are supported:", by add :

- `azurite.tableHost`
- `azurite.tablePort`

File: README.md [hunk 2]
```diff
@@ -97,6 +97,12 @@ Start queue service only:
 $ azurite-queue -l path/to/azurite/workspace
 ```
 
+Start table service only:
```
Comment (blueww) [inline README.md:100]: The above section "## Features & Key Changes in Azurite V3"
Should also include table

**Comment by XiaoningLiu** [comment]
Thanks @ocdi! Can you check @blueww 's comments?

**Comment by ocdi** [comment]
@blueww and @XiaoningLiu I've updated the readme. I am assuming Table also is 2019-12-12 version, and I filled in the support matrix based on supported operations.

**Pull request merged**